e2e/test/scenarios/sharing/public-dashboard.cy.spec.js

```diff
@@ -181,4 +204,22 @@ describe("scenarios > public > dashboard", () => {
     cy.button("Apply").should("not.exist");
     cy.get(".ScalarValue").should("have.text", COUNT_DOOHICKEY);
   });
+
+  it("should only display filters mapped to cards on the selected tab", () => {
+    cy.get("@dashboardId").then(id => {
+      visitPublicDashboard(id);
+    });
+
+    dashboardParametersContainer().within(() => {
+      cy.findByText(textFilter.name).should("be.visible");
+      cy.findByText(numberFilter.name).should("not.exist");
```

Comment:
Should we name something like `unusedFilter` to emphasize it's not used? Like how we're asserting the name "Not Used Filter" (or "Unused Filter") in `e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js`